aries_vcx/Cargo.toml

```diff
@@ -36,6 +38,22 @@ base64 = "0.10"
 openssl = { version = "0.10.35" }
 num-traits = "0.2.0"
 libvdrtools = { git = "https://gitlab.com/left-arm/vdr-tools.git", rev = "debf776c8" }
+# vdrtools alternatives ----
+indy-vdr = "0.3.4"
```

Comment:
I suggest remove feature "zmq_vendored" to fix the android build, something like this:

`indy-vdr = { version = "0.3.4", default-features = false, features = ["ffi", "log"] }`